equinox/_module/_module.py

```diff
@@ -12,119 +11,27 @@
 import jax
 import jax.tree_util as jtu
 import numpy as np
-from jaxtyping import Array, Bool, PyTree
+from jaxtyping import Array, Bool
 
 from .._filters import is_array, is_array_like, is_inexact_array_like
 from .._pretty_print import tree_pformat
 from .._tree import tree_equal
 from ._better_abstract import better_dataclass, BetterABCMeta
 from ._field import field
+from ._flatten import _generate_flatten_functions, _WRAPPER_FIELD_NAMES
```

Comment:
Nit, I use underscores to indicate private-to-file, so this is not an allowed import.

(Probably just remove the underscores from the names.)